### ISSUE TITLE
Better training of generator

### ISSUE THREAD

**Issue by ChristianEschen** [NVlabs/SPADE#61]
Hi

Thanks for the awesome code!
I want to do label to image synthesis using this repository. However, my training data consist of only 16 images.
After 4000 epochs and 4000 epochs with decay I get the following loss curves: (I know this is a lot of epochs).

The discriminator losses
![image](https://user-images.githubusercontent.com/19329210/60077816-18626400-972b-11e9-87aa-c701c599d409.png)

![image](https://user-images.githubusercontent.com/19329210/60077826-1d271800-972b-11e9-8dfa-cda2418ad03d.png)

The generator loss:
![image](https://user-images.githubusercontent.com/19329210/60077835-20ba9f00-972b-11e9-9a62-a763cc39b340.png)

I am not satisfied with the results. After 4000 epochs the generator loss starts to increase.
Does this has to do with the fact that I only have 16 images for training?
If not how can I better train the generator? Could I slow down the discriminator loss?

In advance: Thanks for the help.
Best regards Christian
